<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditROIDialog</class>
 <widget class="QDialog" name="EditROIDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit ROI</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="resolution">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QSpinBox" name="end_y">
     <property name="minimum">
      <number>3</number>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="width">
     <property name="minimum">
      <number>4</number>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QSpinBox" name="height">
     <property name="minimum">
      <number>4</number>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="start_x">
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>End</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QSpinBox" name="start_y">
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Camera Resolution:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Size</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="end_x">
     <property name="minimum">
      <number>3</number>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditROIDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditROIDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
